Add pages-plugin-trpc into Community Pages Plugins (#4486)
cloudflare/cloudflare-docs: content/pages/platform/functions/plugins/community-plugins.md

  Given a folder of assets in multiple formats, this Plugin will automatically negotiate with a client to serve an optimized version of a requested asset.

> - [pages-plugin-trpc](https://github.com/toyamarinyon/cloudflare-pages-plugin-trpc)

Reviewer: ```suggestion
- [cloudflare-pages-plugin-trpc](https://github.com/toyamarinyon/cloudflare-pages-plugin-trpc)
```

Looks like [it's published to npm](https://www.npmjs.com/package/cloudflare-pages-plugin-trpc) under the name `cloudflare-pages-plugin-trpc`.

Reviewer: Other than that (and potentially now re-ordering in the list), LGTM!

Author: Thank you for rewriting!

>     It allows developers to create tRPC server on Cloudflare Page Function rapidly.

Reviewer 2: ```suggestion
    Allows developers to quickly create a tRPC server with a Cloudflare Pages Function.
```
